.gitignore

```diff
@@ -1,9 +1,9 @@
-cnquery
-cnspec
+./cnquery
+./cnspec
```

Comment:
fyi since these were not properly namespaced they caused us to ignore the named folders in apps (which e.g. search utilties started to ignore)